ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -995,6 +1001,41 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
     self.hairline.alpha = alpha;
 }
 
+- (NSArray *)stepsForReviewStep:(ORKReviewStep *)reviewStep {
+    NSMutableArray *steps = [[NSMutableArray alloc] init];
+    if (reviewStep.isStandalone) {
+        for (ORKStep *step in reviewStep.steps) {
+            [steps addObject:step];
+        }
+    } else {
+        ORKStep *nextStep = reviewStep;
```

Comment:
Sounds good. That also means that from now on every standalone review step is presented before it's steps (so `reverseListing` is true), while every embedded review step is presented after it's steps (so `reverseListing` is false). Then we can omit the `_reverseListing` property and use `ORKReviewStep`'s `isStandalone` method instead.